backend/src/apiserver/client_manager.go

```diff
@@ -210,8 +210,8 @@ func initMysql(driverName string, initConnectionTimeout time.Duration) string {
 	mysqlConfig := client.CreateMySQLConfig(
 		getStringConfigWithDefault(mysqlUser, "root"),
 		getStringConfigWithDefault(mysqlPassword, ""),
-		getStringConfig(mysqlServiceHost),
-		getStringConfig(mysqlServicePort),
+		getStringConfigWithDefault(mysqlServiceHost, "mysql"),
```

Comment:
If the config.json is not changed or `getStringConfigWithDefaults` fails and returns "mysql" as the host, will a connection be established?

Reply:
yes! with the current setup, the database is exposed through a service called mysql. Kubernetes provisions pods with the environment variable `MYSQL_SERVICE_HOST` and the mysql service ip as value. Hence the current value for `mysqlServiceHost`. The same service can also be accessed as host `mysql` so the default behavior remains unaltered.